vllm/entrypoints/openai/serving_chat.py

```diff
@@ -176,7 +176,8 @@ async def create_chat_completion(
                 "\"auto\" tool choice requires "
                 "--enable-auto-tool-choice and --tool-call-parser to be set")
 
-        request_id = f"chat-{random_uuid()}"
+        request_id = (f"chat-{request.request_id}" if request.request_id
+                      is not None else f"chat-{random_uuid()}")
```

Comment:
It should be unnecessary to check `request.request_id is not None` because the default is already set by `ChatCompletionRequest`.

Reply:
Good point, changed.